src/test-support/utils.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function transformDetails(options) {
+  const { getTransformType } = require('../transform-support');
+
+  let root = process.cwd() + `/transforms/${options.name}/`;
+  let transformPath = root + 'index.js';
```

Comment:
Seems like this should be using `getTransformPath`

Reply:
ok, I now put `getTransformPath` into `transform-support.js`, and changed the signature a tiny bit, to expect the project's root folder (instead of `binRoot`, as we don't have/need `binRoot` here).